vllm/model_executor/models/gemma3_mm.py

```diff
@@ -610,6 +611,8 @@ def forward(self,
         # NOTE: In v1, inputs_embeds is always generated at model runner, this
         # condition is for v0 compatibility.
         elif inputs_embeds is None:
+            raise AssertionError("hpu_model_runner should be computing ",
+                                 "inputs_embeds")
```

Comment:
remove the line, let's minimize the modeling change, that will be difficult for rebasing and future support with vllm plugin since we can't touch modeling codes.

Reply:
will do